src/components/Atoms/RichText/RichText.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import DOMPurify from 'dompurify';
+
+export const RichTextInner = styled.div`
```

Comment:
RichTextInner could just be Inner or Wrapper .. In styled component we don't need to worry about naming structure .. we can use Wrapper, Inner, Container etc in all components . it will generate random classes

Reply:
Okeydokey, will change to Inner and use that from here onwards for consistency 👍